### DISCUSSION TITLE
Serverless functions?

### DISCUSSION THREAD

**Discussion by VladimirCores** [pocketbase/pocketbase#666] [Q&A] [answered]
Any chance the feature of calling functions on API request will be implemented?

**Comment by ganigeorgiev** [comment]
If you are looking for serverless-style functions support, then no, but there are plans to add simple collection triggers (you could watch issue https://github.com/pocketbase/pocketbase/issues/317).

If you need to execute custom server-side code, you can extend PocketBase with the available [Go event hooks](https://pocketbase.io/docs/event-hooks/).

**Comment by benallfree** [comment, marked as answer]
@VladimirCores I'm working on Typescript/Javascript serverless function support [here](https://github.com/benallfree/pbscript). Is that what you mean?
